.kitchen.yml

```diff
@@ -1,15 +1,14 @@
 ---
 provisioner:
+  name: chef_zero
   data_bag_path: test/integration/data_bags
 
 suites:
   - name: default
+    attributes:
+      poise-python:
+        provider: system
```

Comment:
I think it would make more sense if you set it in the default recipe directly IMO.